NumbersCount.java

```diff
@@ -65,11 +66,122 @@
  */
 
 public class NumbersCount {
+	int[] count = new int[10];
+
+
     public NumbersCount() {
         super();
     }
 
     public static void main(String[] args) {
         NumbersCount numbersCount = new NumbersCount();
+        numbersCount.testBench();
+    }
+    
+    public void printGraph(int[] count){
+    	int column=0;
+    	int rows = count.length;
+    	for(int i=0;i<count.length;i++){
+    		if(count[i]>column){
+    			column = count[i];
+    		}
+    		
+    	}
+	    for(int j =column; j>0;j--){
+
+    	for(int i = 0; i<rows; i++){
+    	        if(count[i]==j){
+    	        	System.out.print("#" + " ");
+    	        	count[i]=count[i]-1;
+    	        }
+    	        else
+    	        	System.out.print(" " + " ");
+    	    }
+    	    System.out.print("\n");
+    	}
+    	for(int i = 0; i<rows; i++){
+    		System.out.print(i + " ");
+    	}
+
+    	
+    }
+    
+    public int[] numberCount(String input){
+    	int index = input.length()-1;
+    	String newInput;
+
+    	if(index<0){
+    		
+    		return count;
+    	}
+    	
+    	if(input.charAt(index)=='0'){
```

Comment:
Good understanding of recursion, nicely implemented, base case was handled properly..
Can we replace the 10 if conditions with just one :) do we really need to check 0-9 separately?